common/web/types/src/kmx/element-string.ts

```diff
@@ -74,6 +74,9 @@ export class ElementString extends Array<ElemElement> {
         typeFlag |= constants.elem_flags_type_uset;
         // TODO-LDML: err on max buffer size
         const needRanges = sections.usetparser.sizeUnicodeSet(item.segment);
+        if (needRanges < 0) {
+          return null; // UnicodeSet error
+        }
```

Comment:
This is the fix.  Note that `sizeUnicodeSet()` already will notify via callback if there's an error. So we can just exit here.

Reply:
```suggestion
        if (needRanges < 0) {
          // Note that sizeUnicodeSet() already will notify via callback if there's an
          // error. So we can just exit here.
          return null; // UnicodeSet error
        }
```